gateway/src/apicast/policy/rate_limiting_to_service/apicast-policy.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+  "name": "rate limiting to service policy",
+  "description": ["This policy adds rate limiting to the entire Service."],
+  "version": "builtin",
+  "configuration": {
+    "type": "object",
+    "properties": {
+      "limitters": {
+        "description": "List of limitters to be applied",
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "limitter": {
+              "description": "Limitter class name",
+              "type": "string"
+            },
+            "key": {
+              "description": "Key of Redis or Name of lua_shared_dict shm zone",
+              "type": "string"
+            },
+            "values": {
+              "description": "Values to be applied",
+              "type": "array",
+              "items": {
+                "type": "integer"
+              }
+            }
+          }
+        }
+      },
+      "redis_info": {
```

Comment:
I think it might be better to use just one property: redis_url.
That can embed authentication, database, port, ...
Wdyt @davidor ?